src/main/java/cam72cam/immersiverailroading/entity/LocomotiveDiesel.java

```diff
@@ -190,11 +204,12 @@ public void onUpdate() {
 					this.soundThrottle += Math.min(0.01f, absThrottle - this.soundThrottle);
 				}
 	
-				if (horn.isPlaying()) {
+				if (this.getDataManager().get(HORN) < 1) {
```

Comment:
Leave this line alone.  The original code was clearer / standard

Reply:
i'll change it back, but for reference, this isn't a peice of the code i touched actually, so not sure how it got changed